src/phoenix/core/traces.py

```diff
@@ -36,17 +36,17 @@ def __init__(self) -> None:
 
     def get_project(self, project_name: str) -> Optional["Project"]:
         with self._lock:
-            return self._projects.get(project_name)
+            return self._projects.get(project_name, Project())
```

Comment:
what's the second parameter for?

Reply:
This is legacy code that will be removed, but for the time being I need to patch it so things don't break as we migrate things incrementally. This particular change is for when the database is out of sync with the memory store in the case of testing but I don't want it to break because it's returning None.